### PR TITLE
Milestone-2-User-interface

### PR DESCRIPTION
**Pull request by sinansevgi**
This is branch contains classes and functions for user input/output actions.

Main.rb :
 - Asks player's name and inputs.
 - Prints game instructions.

Board.rb :
 - Draws game board.

### CODE REVIEW

File: lib/board.rb [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  end
+
+  def legal() end
+
+  def draw
+    # @board.each do |row|
+    #     row.each do |pos|
+    #         puts pos
+    #     end
+    # end
```
Comment (umairarshadbutt) [inline lib/board.rb]: Remove inline comments.

**Review by umairarshadbutt — CHANGES_REQUESTED** [review]
# Status: REQUIRED CHANGES :warning: :construction: 

Hi 
Such a remarkable milestone you made :+1: ! Awesome interface :100: !
We, however, do have a few changes to carry out before we proceed (see review comment).

##

Kindly work on the suggested changes 👇.
- [x] Remove inline comments. ([screenshot](https://user-images.githubusercontent.com/29689273/97425247-a8be4380-1933-11eb-9bde-32ec08effe40.png) from [Pathwright](https://microverse.pathwright.com/auth/sign-in/?next=/library/fast-track-curriculum/69047/path/step/66243643/))

- [x] game informs player if the selected move is a winning move (without computing the winning move) ([screenshot hint](https://user-images.githubusercontent.com/29689273/97425614-2bdf9980-1934-11eb-8186-07c318383f13.png) from [Pathwright](https://microverse.pathwright.com/auth/sign-in/?next=/library/fast-track-curriculum/69047/path/step/59565018/))
- [x] game informs player if the selected move is a draw move (without computing draw)([screenshot hint](https://user-images.githubusercontent.com/29689273/97425614-2bdf9980-1934-11eb-8186-07c318383f13.png) from [Pathwright](https://microverse.pathwright.com/auth/sign-in/?next=/library/fast-track-curriculum/69047/path/step/59565018/))

##
- **When you have made the specified changes, please submit another code review. Thank you!**
##
**If you are stuck at any point or have any questions, let me know by leaving a comment here and mentioning me. I will be happy to help**
HAPPY CODING :computer:
**[Umair Arshad](https://umairarshad.me/)**
[Twitter](https://twitter.com/its_UmairArshad) | [LinkedIn](https://www.linkedin.com/in/umair-arshad-butt/) | [GitHub](https://github.com/umairarshadbutt)

**Review by layzrd — CHANGES_REQUESTED** [review]
### Status: Require changes 🌳 ♲
Hi @bini-i  and @sinansevgi, 👏 You did very well on this project. Most requirements have done 👌, Kindly fix the following issues.

[REQUIRED]

- [x] The game should inform the player if the selected move is invalid (without computing what an invalid move is)
- [x] The game should display the board after the player move (without computing which spots on the board are taken)
- [x] The game should inform the player if the selected move is a winning move (without computing the winning move)
- [x] The game should inform the player if the selected move is a draw move (without computing draw)
- [x]  The game should repeat all actions for the next player's move

Once you fix the issues, kindly submit another code review ♲
 Happy coding 👨‍💻
 [Abdulaziz Redi Ali](https://azizali.ml/)

**Review by arslanbisharat — APPROVED** [review]
# Approved :smile: 

Hello Developer, :wave:

I just reviwed your project and I am very happy to annocunce that your project is approved.
Congraulation for achieving this milestone towards your career. :clap:

_I am looking forward towards to see your next project._

-----------------------------------------------------------------------------------------------

**Regards,
Arslan Bisharat
slack: arslanbisharat**

**Pull request merged**